packages/lwc-language-server/src/tag.ts

```diff
@@ -117,12 +117,13 @@ export default class Tag implements ITagData {
     }
 
     classMember(name: string): ClassMember | null {
-        return this.classMembers.find(item => item.name === name) || null;
+        return this.classMembers?.find(item => item.name === name) || null;
     }
 
     classMemberLocation(name: string): Location | null {
         const classMember = this.classMember(name);
         if (!classMember) {
+            console.log(`Attribute "${name}" not found`);
```

Comment:
@sbudhirajadoc this is the user-facing text that will be shown in the Output tab when the LWC Language Server option is selected.